### ISSUE TITLE
Color contrast accessibility 

### ISSUE THREAD

**Issue by mgifford** [evidence-dev/evidence#854]
**Bug Description**
Most pages I've looked at have color contrast errors.

**Severity**

- [x] High - I'm not able to use Evidence until this bug is fixed
- [ ] Medium - I'm able to use Evidence, but a feature I need to use is not working
- [ ] Low - I'm able to keep using Evidence as normal, but flagging this for the team to fix

**Expected Behavior**

1 error would be no big deal but it is everywhere.  I've just put in one example.

The problem with having accessibility problems in open source projects is that they spread. 

Good accessibility defaults are important, because they replicate. 

**Error Messages and Screenshots**
**Steps To Reproduce**

Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.page-count)
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: Evidence - https://business-review-demo.netlify.app/

Element path: .page-labels > .page-count

Snippet: <span class="page-count svelte-1g0oohe">Page <input class="page-input svelte-1g0oohe" type="number" placeholder="1"> / <span class="page-count svelte-1g0oohe" style="margin-left: 4px;">16</span></span>

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 3.7 (foreground color: #7b8693, background color: #ffffff, font size: 9.0pt (12px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Microsoft Edge version 112.0.1722.68

====

This accessibility issue was found using Accessibility Insights for Web 2.38.1 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by mcrascal** [comment]
Hi Mike, thanks for flagging this! We'll take a look, we'd also welcome a PR if you're interested in contributing!

**Comment by mgifford** [comment]
I'm just looking at the project. Haven't installed it yet.  My main accessibility contributions are on Drupal.org.
